repository-data/development/src/main/resources/hcm-content/content/documents/lks/landing-pages.yaml

```diff
@@ -1,9 +1,9 @@
-/content/documents/lks/listing-pages:
```

Comment:
Could we add an Content Action [`hcm-actions.yaml`] in order to get rid of existing `listing-pages` folder and it's descendants from local/brCloud dev environments ?

Reply:
@smanivasag I have updated that.

Reply:
Thanks @thojava